recipe/meta.yaml

```diff
@@ -62,6 +62,7 @@ requirements:
     - pyyaml
     - m2-git  # [win]
     - git     # [unix]
+    - cctools 949.*  # [osx]
```

Comment:
New ones don't work at the moment. See https://github.com/tpoechtrager/cctools-port/pull/105#issuecomment-908459824

Reply:
We should add that as a comment so we don't forget why. Otherwise LGTM 👍 
```suggestion
    # New ones don't work at the moment. See tpoechtrager/cctools-port#105 (comment)
    - cctools 949.*  # [osx]
```